### PR TITLE
feat: tcl aura new pid migration

### PR DESCRIPTION
**Pull request by petrovska-petro**
while waiting for the avatar dd we will migrate liq directly from the vault msig

```
brownie run badger/aura_liq_migration
```

### CODE REVIEW

**Review by sajanrajdev — DISMISSED** [review]
## Review
- Confirmed new PIDs and new rewards contracts:
image
image
- Confirmed new migrator contract, deployed 2 weeks 
- Migration script properly asserts balance persistence
- Approvals only performed on the exact amount staked for each asset
- Script runs properly and events confirm proper functioning 

LGTM

File: helpers/addresses.py [hunk 1]
```diff
@@ -624,17 +624,19 @@
...
-        "aura_40wbtc40digg20gravi_gauge": "0xD7c9c6922db15F47EF3131F2830d8E87f7637210",
+        "aura_40wbtc40digg20gravi_gauge": "0x80e355454b608d43402147834d05f424b6f706b8",
         "aura_50reth50badger": "0x685C94e7DA6C8F14Ae58f168C942Fb05bAD73412",
         "aura_50badger50reth_gauge": "0x4E867c6c76173539538B7a9335E89b00434Aec10",
         "aura_50reth50badger_gauge": "0xAAd4eE162Dbc9C25cCa26bA4340B36E3eF7C1A80",
         "_deprecated": {
             "aura_20wbtc80badger": "0xCea3aa5b2a50e39c7C7755EbFF1e9E1e1516D3f5",
             "aura_40wbtc40digg20gravi": "0x10Ca519614b0F3463890387c24819001AFfC5152",
+            "aura_20wbtc80badger_2nd_migration": "0x4EFc8DED860Bc472FA8d938dc3fD4946Bc1A0a18",
+            "aura_40wbtc40digg20gravi_gauge_2nd_migration": "0xD7c9c6922db15F47EF3131F2830d8E87f7637210",
```
Comment (gosuto-inzasheru) [inline helpers/addresses.py:639]:
```suggestion
            "aura_40wbtc40digg20gravi_gauge_2nd_migration": "0xD7c9c6922db15F47EF3131F2830d8E87f7637210",
            "gauge_migrator": "0x7954bcDce86e86BeE7b1dEff48c3a0b9BCCe578B",
```

**Review by gosuto-inzasheru — DISMISSED** [review]
- new gauge addresses confirmed
- pids confirmed through ui: https://app.aura.finance/#/1/pool/111
- assert balance from old to new gauge doesnt change
- runs good to me

**Review by gosuto-inzasheru — APPROVED** [review]

**Review by sajanrajdev — APPROVED** [review]
Changes LGTM

**Pull request merged**